src/test/java/com/reckue/post/controllers/RatingControllerIntTest.java

```diff
@@ -98,48 +98,11 @@
 //        List<RatingResponse> expected = ratingRepository.findAll().stream()
 //                .map(RatingConverter::convert)
 //                .sorted(Comparator.comparing(RatingResponse::getId).reversed())
-//                .collect(Collectors.toList());
-//
-//        List<RatingResponse> actual = objectMapper
-//                .readValue(this.mockMvc.perform(get("/ratings?desc=true&limit=2&offset=0&sort=id"))
-//                        .andDo(print())
-//                        .andExpect(status().isOk())
-//                        .andReturn()
-//                        .getResponse().getContentAsString(), new TypeReference<>() {
-//                });
-//
-//        Assertions.assertEquals(expected, actual);
-//    }
-//
-//    @Test
-//    public void findAllSortedByTextAsc() throws Exception {
-//        List<RatingResponse> expected = ratingRepository.findAll().stream()
-//                .map(RatingConverter::convert)
-//                .sorted(Comparator.comparing(RatingResponse::getId))
-//                .limit(2)
-//                .collect(Collectors.toList());
-//
-//        List<RatingResponse> actual = objectMapper
-//                .readValue(this.mockMvc.perform(get("/ratings?desc=false&limit=2&offset=0&sort=text"))
-//                        .andDo(print())
-//                        .andExpect(status().isOk())
-//                        .andReturn()
-//                        .getResponse().getContentAsString(), new TypeReference<>() {
-//                });
-//
-//        Assertions.assertEquals(expected, actual);
-//    }
-//
-//    @Test
-//    public void findAllSortedByTextDesc() throws Exception {
-//        List<RatingResponse> expected = ratingRepository.findAll().stream()
-//                .map(RatingConverter::convert)
-//                .sorted(Comparator.comparing(RatingResponse::getId).reversed())
 //                .limit(2)
 //                .collect(Collectors.toList());
 //
 //        List<RatingResponse> actual = objectMapper
-//                .readValue(this.mockMvc.perform(get("/ratings?desc=true&limit=2&offset=0&sort=text"))
+//                .readValue(this.mockMvc.perform(get("/ratings?desc=true&limit=2&offset=0&sort=id"))
```

Comment:
You need to correct url "ratings" to "rating".
As well as you need to delete in class RatingConverter line 26 ".id(ratingRequest.getId())" from public static Rating convert(RatingRequest ratingRequest).
And your tests will be passed!